install/install.go

```diff
@@ -441,9 +441,13 @@ func resolveDst(dst string) string {
 }
 
 func cleanOld(state *client.GooGetState, pi goolib.PackageInfo, insFiles map[string]string, dbOnly bool) {
+	// TODO: Use error wrapping here https://blog.golang.org/go1.13-errors
 	st, err := state.GetPackageState(pi)
 	if err != nil {
-		return
+		if strings.Contains(err.Error(), "no match found for package") {
+			return
+		}
+		logger.Error(err)
```

Comment:
You should return at the end of this no matter what otherwise you will have issues further down (state var will be empty).
The easiest would be to just:
```
if !strings.Contains(err.Error(), "no match found for package") {
  logger.Error(err)
}
return
```

Reply:
done